tools/fileinfo/bugs/issue-460-hash-from-empty-string/test.py

```diff
@@ -11,6 +11,6 @@ def test_for_all_needed_libs(self):
         assert self.fileinfo.succeeded
 
         imports = self.fileinfo.output['importTable'];
-        assert 'crc32' not in imports
-        assert 'md5' not in imports
-        assert 'sha256' not in imports
+        assert 'crc32' in imports
```

Comment:
This was a test to check the solution to https://github.com/avast/retdec/issues/460 as described in https://github.com/avast/retdec/issues/460#issuecomment-505900997.

The old solution may not have been correct. The new solution in https://github.com/avast/retdec/pull/1003 may be better, but:
1. Is it really better now? For the sample in this test, and also regarding the situation described by @s3rvac in the original issue? I.e. wasn't this test modified just to work with the new implementation?
2. If it is better now, please comment on the original issue https://github.com/avast/retdec/issues/460, and also do something (I'm not sure what) with this test. It looks like it wanted to test that hashes are not computed, so if they are, and they should be, either we should remove the test (probably not), or test the values or something (probably better) - not just that there are some, but untested, hashes.

Reply:
I reverted the original behavior in https://github.com/avast/retdec/commit/3d4a6ff4577170e22067d240d616405b91d7f421.